src/processor_arm.cpp

```diff
@@ -1818,9 +1819,15 @@ jl_image_t jl_init_processor_pkgimg(void *hdl)
     return parse_sysimg(hdl, pkgimg_init_cb);
 }
 
-JL_DLLEXPORT void jl_check_pkgimage_clones(char *data)
+JL_DLLEXPORT jl_value_t* jl_check_pkgimage_clones(char *data)
 {
-    pkgimg_init_cb(data);
+    jl_value_t *rejection_reason;
```

Comment:
```suggestion
    jl_value_t *rejection_reason = NULL;
```
(you can't root undef)